devlab/config.template

```diff
@@ -113,6 +114,7 @@ ext_cidr = <dst_ext_cidr>
 user = <dst_user>
 password = <dst_password> 
 tenant = <dst_tenant>
+service_tenant = services
```

Comment:
That's true for Fuel/MOS, but not for the devlab we have. In devlab we use 'service' everywhere.